### PR TITLE
feat: adding vite to the existing app

### LINKED ISSUE #13: [FEATURE] Adding vite to the existing app
Currently we are using the good old `create-react-app` however that is not recommended at all. It would be much faster if we could integrate `Vite` to it. Here's a guide for the same.

[Adding vite to react webapps](https://joshuaclaudioenrico.medium.com/why-vite-and-how-to-migrate-a-existing-react-aplication-cd1b38d10220), please use this guide for the same.

### PR DESCRIPTION
**Pull request by sayakongit**
## Description

This pull request migrates the current Create-React-App to Vite for the better performance of the site.

## Related Issue(s)

Fixes #13

## Changes Made

1. Installed vite and vite plugins
2. Removed react-scripts from `package.json` and added vite scripts instead.
3. Added `vite.config.js` file
4. Changed `.js` files to `.jsx`
5. Removed `%PUBLIC_URL%` in `index.html`
6. Fixed a import warning in [InfiniteCarousel.css](https://github.com/hack4bengal/s3/compare/main...sayakongit:s3:cra-to-vite?expand=1#diff-0593d1302fdc65f19e21c06f4e627b0e565f6b843e34a36ade4f753d8d3096aa)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,27 +8,30 @@
...
     "web-vitals": "^2.1.4"
   },
   "scripts": {
     "predeploy": "npm run build",
     "deploy": "gh-pages -d build",
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "start": "vite",
```
Comment (tamalCodes) [inline package.json]:
```suggestion
    "dev": "vite --port 3000",
```

**Review by tamalCodes — CHANGES_REQUESTED** [review]

**Comment by tamalCodes** [comment]
@imanishbarnwal , take a look at the changes here, I guess this would require us to change the vercel settings from `npm start` to `npm run build` ?

Also @sayakongit , please fix the conflicts.

**Comment by sayakongit** [comment]
@tamalCodes Fixed the conflicts! Can you check once now?

**Comment by sayakongit** [comment]
@tamalCodes Can you please look into this and let me know if any more changes are required?

**Pull request merged**